lib/Ast.ml

```diff
@@ -198,6 +198,16 @@ module String_id = struct
     && Option.is_none
          (String.lfindi s ~pos:3 ~f:(fun _ c -> not (Char_id.is_kwdop c)))
 
+  let is_monadic s =
+    let is_monadop s =
+      match String.sub s ~pos:0 ~len:(min 2 (String.length s)) with
+      | ">>" | ">|" | "@@" | "@>" -> true
+      | _ -> false
+    in
+    is_monadop s
+    (* "*>>=", "+>>>", "/>>|", etc. *)
+    || (String.length s > 3 && is_monadop (String.sub s ~pos:1 ~len:2))
```

Comment:
It seems that you really want to check whether `s` contains `">>"`. Do you have a list of the operators that should be considered a monadic bind ?

Reply:
I just used the same definition as `ocp-indent` (https://github.com/OCamlPro/ocp-indent/blob/master/src/indentBlock.ml#L440-L450) for consistency.

Reply:
ocp-indent does that to indent differently some operators. OCamlformat doesn't do that, it formats specifically the `_ OP fun _ -> EXP` case. (and even fore specifically when `EXP` is an operator apply)
To be more consistent with ocp-indent you would need to format every operators that way (not just when it's followed by a fun), which would be a big change. 
I don't think we need consistency with ocp-indent here.

Reply:
I reverted this change, on the other hand what do you think of always breaking after the `->` ? if not then we can close this PR.

Reply:
Breaking after the `->` isn't nice if the expression fits on one line. For example:
```
  f (g >>= fun _ -> h)
```
Perhaps breaking after the `->` if there's a chain of more than one operator ?